common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftClientResource.java

```diff
@@ -2,13 +2,18 @@
 
 import io.fabric8.openshift.client.DefaultOpenShiftClient;
 import io.fabric8.openshift.client.OpenShiftClient;
+import io.fabric8.openshift.client.OpenShiftConfig;
+import io.fabric8.openshift.client.OpenShiftConfigBuilder;
 import org.junit.jupiter.api.extension.ExtensionContext.Store.CloseableResource;
 
 final class OpenShiftClientResource implements CloseableResource {
     final OpenShiftClient client;
 
     static OpenShiftClientResource createDefault() {
-        return new OpenShiftClientResource(new DefaultOpenShiftClient());
+        OpenShiftConfig config = new OpenShiftConfigBuilder()
+                .withTrustCerts(true)
```

Comment:
So this is equivalent to `quarkus.kubernetes-client.trust-certs=true`, right?

Reply:
Yes.

Reply:
Indeed:

https://github.com/quarkusio/quarkus/blob/53d41bd92b127d3ecc76f9678414343696ded7c8/extensions/kubernetes-client/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientUtils.java#L53

Reply:
I mean, the commit message is misleading :-)

Reply:
Should @mjurc amend it?

Reply:
I've updated it :)